test/common/stats/stats_impl_test.cc

```diff
@@ -477,15 +477,6 @@ TEST(TagProducerTest, CheckConstructor) {
       "No regex specified for tag specifier and no default regex for name: 'test_extractor'");
 }
 
-// Validate truncation behavior of RawStatData.
-TEST(RawStatDataTest, Truncate) {
```

Comment:
Do we have an equivalent test that tests the truncation wherever it happens now?

Reply:
Yes, HotRestartImplTest.truncateKey also tests this, but I'll put back in a native RawStatDataTest.Truncate test.